### ISSUE TITLE
Expose original unmodified protocol methods through another object

### ISSUE THREAD

**Issue by vvo** [webdriverio/webdriverio#130]
With the new behaviors of `.element` or `click`, we broke original protocol methods.

Thus, people wanting to only use original protocol methods, can't.

We should have a sub-namespace holding all protocol methods and exposing this sub-namespace in the documentation.

Right now, we are documenting `buttonPress` as `click` protocol because we use click as our "super handler".

And `element` can't be used the way it could be used previously (thus, we introduced an API change and we may have broke many clients if they update).

tl;dr; provide every selenium protocol methods, untouched, in another namespace.

Use that namespace from our helper functions.

@christian-bromann @ysono, what do you think?
